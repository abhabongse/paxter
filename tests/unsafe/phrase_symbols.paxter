@python##"
    _symbols_ = {
        '@': '@',
        '.': '&hairsp;',
        ',': '&thinsp;',  # approx 3/18 M
        ':': '&emsp14;',  # approx 4/18 M
        ';': '&emsp13;',  # approx 5/18 M
    }
    ensp = '&ensp;'
    emsp = '&emsp;'
"##\
The final exam is between 13:00@.â€“@.16:00.@"\"
Hello.
