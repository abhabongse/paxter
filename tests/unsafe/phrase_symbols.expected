The final exam is between 13:00&hairsp;â€“&hairsp;16:00.\
Hello.
